var/spack/repos/builtin/packages/py-sympy/package.py

```diff
@@ -11,6 +11,7 @@ class PySympy(PythonPackage):
 
     pypi = "sympy/sympy-0.7.6.tar.gz"
 
+    version("1.13.0", sha256="3b6af8f4d008b9a1a6a4268b335b984b23835f26d1d60b0526ebc71d48a25f57")
```

Comment:
From upstream?

Reply:
Downloaded the package from PyPI and did `sha256sum sympy-1.13.0.tar.gz`. The numbers for previous versions match this procedure, so I suppose that's the right thing to do?

Reply:
Easier: `spack checksum <package>`, and we should make sure that modifications to `builtin` packages make their way upstream. Otherwise, we're having a lot of fun with new deployments.

Reply:
Should I open a PR upstream as well then (for sympy, that is)?

Reply:
Would be very nice! Maybe check the dependencies for updates (they will otherwise)

Reply:
Submitted upstream at: https://github.com/spack/spack/pull/46163

Reply:
Would you mind adding the dependencies from there here too?